src/App.jsx

```diff
@@ -14,33 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortByParameter(goods, { param = '' }) {
+  let goodsArr = goods;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (param) {
+    case 'alphabetically': {
+      goodsArr = goods.sort((good1, good2) => {
+        return good1.localeCompare(good2);
+      });
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+      break;
+    }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    case 'length': {
+      goodsArr = goods.sort((good1, good2) => {
+        return good1.length - good2.length;
+      });
+
+      break;
+    }
+
+    default: {
+      break;
+    }
+  }
+
+  return goodsArr;
+}
+
+function getGoodsFormatted(goods, reversed) {
```

Comment:
you can merge `getGoodsFormatted ` with `sortByParameter `
You will get the function like this
```
function getSortedGoods(goods, sortField, isReversed) {
  const preparedGoods = [...goods];

  if (sortField) {
    preparedGoods.sort((good1, good2) => {
      switch (sortField) {
        case SortType.ALPHABETICALLY:
          {your_logic}

        case SortType.LENGTH:
          {your_logic}

        default:
          {your_logic}
      }
    });
  }

  return isReversed ? preparedGoods.reverse() : preparedGoods;
}
```